modules/nw-multus-remove-pod.adoc

```diff
@@ -3,86 +3,26 @@
 // * networking/multiple_networks/removing-pod.adoc
 
 [id="nw-multus-remove-pod_{context}"]
-= Removing a Pod from an additional network
+= Removing a pod from an additional network
 
-You can remove a Pod from an additional network.
+A pod can be removed from an additional network only by deleting the pod.
 
 .Prerequisites
 
-* You have configured an additional network for your cluster.
-* You have an additional network attached to the Pod.
+* An additional network attached to the pod.
```

Comment:
If the rest of the prereqs are sentences, maybe try to make this one be a sentence too. "An additional network has been attached to the pod."?